tests/test_addon_args.py

```diff
@@ -0,0 +1,28 @@
+"""Tests addon argument parsing."""
+
+import sh
+import pytest
+import yaml
+
+
+def test_invalid_addon():
+    with pytest.raises(sh.ErrorReturnCode_1):
+        sh.microk8s.enable.foo()
+
+
+def test_help_text():
```

Comment:
Please add this in the test_addons suite so we run the test eitherway.

Reply:
I moved it over to `test-addons.py`, but am marking it as `addon_args` so that we can run it separately. I think we'd want to run these tests against a blank-slate MicroK8s install, otherwise addons may be in unusual states that will break these tests.